models/User.model.js

```diff
@@ -2,12 +2,19 @@ const { Schema, model } = require("mongoose");
 
 // TODO: Please make sure you edit the user model to whatever makes sense in this case
 const userSchema = new Schema({
-  username: {
+  email: {
     type: String,
-    unique: true
+    unique: true,
+    required: [true, 'Email is required'],
+    match: [/^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/, 'Please enter a valid email'],
```

Comment:
🔥  this is soo cool to get sure that is an email the thing that you are saving in your DB